juriscraper/oral_args/united_states/federal_appellate/ca9.py

```diff
@@ -3,45 +3,28 @@
 Court Short Name: ca9
 """
 
-from juriscraper.AbstractSite import logger
-from juriscraper.lib.html_utils import (
-    get_row_column_links,
-    get_row_column_text,
-)
-from juriscraper.OralArgumentSiteLinearWebDriven import (
-    OralArgumentSiteLinearWebDriven,
-)
+from juriscraper.lib.html_utils import get_row_column_text
+from juriscraper.OralArgumentSiteLinear import OralArgumentSiteLinear
 
 
-class Site(OralArgumentSiteLinearWebDriven):
+class Site(OralArgumentSiteLinear):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.court_id = self.__module__
         self.url = "https://www.ca9.uscourts.gov/media/"
-        self.uses_selenium = True
-
-    def _download(self, request_dict={}):
-        if self.test_mode_enabled():
-            return super()._download(request_dict)
-        else:
-            self.initiate_webdriven_session()
-            return super()._download(request_dict)
 
     def _process_html(self):
         path = "//table[@id='search-results-table']//tr"
         rows = self.html.xpath(path)
-
-        for row in rows:
-            # In the last row there is not valid data, but a one celled table footer
-            cells = row.xpath(".//td")
-            if len(cells) != 7:
-                continue
-            # Skip cases where the URL includes spaces
-            url = get_row_column_links(row, 6)
-            if " " in url:
-                logger.warning(f"URL includes spaces: {url}")
-                continue
-
+        for row in rows[1:-2]:
+            parts = row.xpath(".//td[6]/a/@href")[0].split("/")
+            year = parts[-3][1:5]
+            month = parts[-3][5:7]
+            day = parts[-3][7:]
+            docket_number = parts[-2]
```

Comment:
Yowza. Can you explain what all this indexing is doing for the poor sap that comes next?

Reply:
yessir.

Reply:
back to you

Reply:
It's always helpful to have an example as well as an explanation, but this helps. Merging.